src/bp/core/database/tables/bot-specific/conversations.ts

```diff
@@ -0,0 +1,19 @@
+import { Table } from 'core/database/interfaces'
+
+export class ConversationsTable extends Table {
+  readonly name: string = 'conversations'
+
+  async bootstrap() {
+    let created = false
+
+    await this.knex.createTableIfNotExists(this.name, table => {
+      table.increments('id').primary()
+      table.string('userId') // .references('id').inTable('users')
```

Comment:
why not add .references('id') right away ?

Reply:
Both tables that contains users at the moment have the user linked to a channel so it wouldn't make sense to reference it in a conversation that is meant to be channel neutral